src/protocol/utils.ts

```diff
@@ -72,16 +70,9 @@ function NotAllowed() {
 }
 
 export const DisallowEverythingProxyHandler: ProxyHandler<object> = {
-  // getPrototypeOf() {
-  //   NotAllowed();
-  //   return null;
-  // },
-  //has() {
-  //  NotAllowed();
-  //  return false;
-  //},
   get() {
-    NotAllowed();
+    return null;
+    // NotAllowed();
```

Comment:
I'm not sure about this one but I don't know the consequence of it.

cc: @loganfsmyth  (I think you were already mentioned on this so apologies if this is duplicate).

Reply:
I'd `return undefined;` instead of `null`, but otherwise I think this change is enough that changing the the `Object.create(null)` calls back to `{}` will be fine.